iosApp/PhongKMMIC/Sources/Presentation/Modules/Home/Views/SurveyContentView.swift

```diff
@@ -0,0 +1,159 @@
+//
+//  SurveyContentVIew.swift
+//  PhongKMMIC
+//
+//  Created by Phong Vo on 07/05/2023.
+//  Copyright © 2023 Nimble. All rights reserved.
+//
+
+import SwiftUI
+
+struct SurveyContentView: View {
+
+    var uiModels: [UIModel]
+    @State private var currentPage = 0
+    @State private var nextPage = 0
+    @State private var currentVisibility: Double = 1.0
+
+    @DragGestureVelocity private var velocity: CGVector
+
+    private var draggingSpeed: Double = 700.0
+    private var draggingLength: Double = 160.0
+    private var draggingVisibilityMultiplier: Double = 1.2
+
+    var body: some View {
+        // swiftlint:disable:next closure_body_length
```

Comment:
Please separate this body into small views, this rule is good to notify us when we need to chop this body into small ones 🙏